src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/InsertOperation.java

```diff
@@ -195,7 +208,9 @@ private static SimpleStatement bindInsertValues(
           CQLBindValues.getDoubleMapValues(doc.queryNumberValues()),
           CQLBindValues.getStringMapValues(doc.queryTextValues()),
           CQLBindValues.getSetValue(doc.queryNullValues()),
-          CQLBindValues.getTimestampMapValues(doc.queryTimestampValues()),
+          offlineMode
```

Comment:
Should have a note on why difference needed? (cannot unify for some reason)

Reply:
Added comment in this commit [9f2dc01](https://github.com/stargate/data-api/pull/905/commits/9f2dc01ab7a78ceedb05387a8c34f2febed0fb19)